### PR TITLE
Design of single list entity

### PR DESCRIPTION
**Pull request by Kori-Kozicki**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml
Comment (SuperCreativeUniqueUsername) [inline app/src/main/AndroidManifest.xml:1]: what does line 24-26 do?

Comment (Kori-Kozicki) [reply app/src/main/AndroidManifest.xml:1]: It must've automatically got added when I imported the roboto fonts to use. Otherwise I'm not sure

**Comment by ghost** [comment]
The code is causing some errors on my end in oEntrantArrayAdapter with 
        accountName.setText(account.getfirstName() + " " + account.getlastName());
        accountPhone.setText(account.getPhone());
        accountEmail.setText(account.getEmail());
        accountStatus.setText(account.getStatus());

I think the get method names might need to be updated?

**Comment by Kori-Kozicki** [comment]
> The code is causing some errors on my end in oEntrantArrayAdapter with accountName.setText(account.getfirstName() + " " + account.getlastName()); accountPhone.setText(account.getPhone()); accountEmail.setText(account.getEmail()); accountStatus.setText(account.getStatus());
> 
> I think the get method names might need to be updated?

Yah I was doing some draft work and didn't know my recent commits got added to this pull request too lol. I'll have this fixed tn and have something better

**Comment by SuperCreativeUniqueUsername** [comment]
I think it would make sense to connect this to what Jen made for the navigation bar by merging the main branch onto your branch and modifying it, just for consistency. This is just an idea you don't need to do what I'm saying. It might make sense to close this PR and submit a new one once its ready for review again.

**Comment by Kori-Kozicki** [comment]
> I think it would make sense to connect this to what Jen made for the navigation bar by merging the main branch onto your branch and modifying it, just for consistency. This is just an idea you don't need to do what I'm saying. It might make sense to close this PR and submit a new one once its ready for review again.

Yah I already got that set up. Rn it navigates to my work when you change the role in profile to organizer (except I haven't pushed that yet). Honestly I've been considering closing the pull request and then making a new one once everything is set up.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -9,5 +9,11 @@
         <item>Organizer</item>
         <item>Admin</item>
     </string-array>
+    <string-array name="entrant_status">
```
Comment (katzhong) [inline app/src/main/res/values/strings.xml:12]: i think we might also need to include a pending (when organizer hasn't made decision yet), lost (entrant loses initial lottery), and declined status (entrant doesn't want to go after being accepted). also just curious, what do the waitlist and invited status mean?

Comment (Kori-Kozicki) [reply app/src/main/res/values/strings.xml:12]: I guess it's kinda vague but those statuses you're describing are what is there. On waitlist means "pending" as there's been no decision yet (Just every entrant who has joined the waitlist) and canceled is the declined status. They canceled their entry. Invited just means that they won the lottery and they've recieved a notification to join the event. Also I don't think in the user stories we need to worry about who lost the lottery

**Comment by SuperCreativeUniqueUsername** [comment]
> > I think it would make sense to connect this to what Jen made for the navigation bar by merging the main branch onto your branch and modifying it, just for consistency. This is just an idea you don't need to do what I'm saying. It might make sense to close this PR and submit a new one once its ready for review again.
> 
> Yah I already got that set up. Rn it navigates to my work when you change the role in profile to organizer (except I haven't pushed that yet). Honestly I've been considering closing the pull request and then making a new one once everything is set up.

I think it could make sense to close the pr until you're happy with it. Otherwise, could you just tell us when you're ready for us to review?

**Comment by Kori-Kozicki** [comment]
> > > I think it would make sense to connect this to what Jen made for the navigation bar by merging the main branch onto your branch and modifying it, just for consistency. This is just an idea you don't need to do what I'm saying. It might make sense to close this PR and submit a new one once its ready for review again.
> > 
> > 
> > Yah I already got that set up. Rn it navigates to my work when you change the role in profile to organizer (except I haven't pushed that yet). Honestly I've been considering closing the pull request and then making a new one once everything is set up.
> 
> I think it could make sense to close the pr until you're happy with it. Otherwise, could you just tell us when you're ready for us to review?

Yah I'm just gonna do that. This has been sitting in here too long tbh

**Pull request closed without merging**